etc/kayobe/kolla/kolla-build.conf

```diff
@@ -9,3 +9,8 @@ base_tag = jammy-20231004
 base_tag = 9.{{ stackhpc_pulp_repo_rocky_9_minor_version }}
 {% endif %}
 build_args = {{ kolla_build_args.items() | map('join', ':') | join(',') }}
+
+[openstack-base]
+type = url
+location = https://github.com/stackhpc/requirements
+reference = {{ stackhpc_kolla_source_version }}
```

Comment:
We may decide to start using tags for the kolla version, at which point this would be wrong. It would be better to either make new variables for the requirements URL and version, or just hard code to `stackhpc/{{ openstack_release }}`